sql/core/src/test/resources/test-data/with-array-fields

```diff
@@ -0,0 +1,3 @@
+{ "id": 1, "intervals": [{ "startTime": 111, "endTime": 211 }, { "startTime": 121, "endTime": 221 }], "values": [11, 21, 31, 41]}
```

Comment:
Could you rename this to `with-array-fields.json`?

Reply:
Sure.